Documentation/persistentvolume-metrics.md

```diff
@@ -0,0 +1,7 @@
+# PersistentVolumeClaim Metrics
+
+| Metric name| Metric type | Labels/tags |
+| ---------- | ----------- | ----------- |
+| kube_persistentvolume_status_phase | Gauge | `persistentvolume`=&lt;pv-name&gt; <br> `namespace`=&lt;pv-namespace&gt; <br>`phase`=&lt;Bound\|Failed\|Pending\|Available\|Released&gt;<br>`volume`=&lt;pv-namespace&gt;|
```

Comment:
One last nit: There should be no `volume` label for `kube_persistenvolume_status_phase`.